docs/markdown/Keyval-module.md

```diff
@@ -23,7 +23,7 @@ chosen the configuration options), output a ".config" file.
 The module may be imported as follows:
 
 ``` meson
-keyval = import('unstable-keyval')
+keyval = import('keyval')
```

Comment:
I would like to take this opportunity to change the import implementation to allow both, so projects using "unstable-keyval" won't stop working.

Reply:
@xclaesse good idea, see my additional commit. I wonder if I should make it a different MR though.